### PR TITLE
First release version

### PR DESCRIPTION
**Pull request by Zappat0n**
- Add npm
- Add Webpack
- Add basic Html skeleton
- Add different JavaScript modules to inject Html content
- Add assets
- Add styling
- Create a production-ready project in the dist folder
- Add README.md

### CODE REVIEW

File: src/tab.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    container.innerHTML = '';
+    addTitle(obj.title);
+    addImage(obj.image);
+    obj.content.forEach(element => {
+      addArticle(element);
+    });
+  };
+
+  return { run };
+}
```
Comment (peterrobert) [inline src/tab.js]: Kindly you should use the ES6 arrow functions. 
es6_requirement (1)

File: src/tab_contact.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    content: [
+      {
+        title: 'Book a table',
+        body: 'If you want to book a table please call: 555-9860-641\nYou can also do it by email: mylittletavern@doodle.com',
+        date: 'We are looking forward to meet you!',
+      },
+    ],
+  };
+  tabFactory(obj).run();
+}
```
Comment (peterrobert) [inline src/tab_contact.js]: Kindly use the ES6 arrow functions

File: src/tab_menu.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    content: [
+      {
+        title: 'We have added the following winter dishes to our menu',
+        body: 'Classic Chicken Pot Pie\nFridge Clean-Out Nabe With Mushroom Dashi\nCreamy-ish of Mushroom Soup\nBrothy Tomato and Fish Soup With Lime',
+        date: 'We are sure you will enjoy them!',
+      },
+    ],
+  };
+  tabFactory(obj).run();
+}
```
Comment (peterrobert) [inline src/tab_menu.js]: Kindly use ES6 arrow functions

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED ♻️ ⏹️ 
Hello, @Zappat0n  Very nice project. I love your design. It is simple and functional. Also great work with your commit messages, they are capitalized and they follow best practice. 👏🏿 

However, there are a few changes that need to be addressed kindly  check the following:

- [ ] In each of your tabbed modules,  you should have something like this:

`const myModule = (parentNode) => {
  const node = document.createElement("div");
  // Add whichever sub-nodes, ids and style is required

  parentNode.appendChild(node);
};

export { myModule as default };
`
Each of your modules should create a div element. In your project, you have created objects which is not the approach described here ⬇️ 

tab_requirement

### if you have any question kindly reach me on slack @peterrobert

### HAPPY CODING 💻

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi @Zappat0n :wave: 

Go on with your next task.

### Have a great day :slightly_smiling_face: 


#### ``` Uchechukwu Henry Inyama ```

**Pull request merged**